src/evidently/options/data_drift.py

```diff
@@ -16,7 +16,14 @@ class DataDriftOptions:
     drift_share: float = 0.5
     nbinsx: Union[int, Dict[str, int]] = DEFAULT_NBINSX
     xbins: Optional[Dict[str, int]] = None
+
     feature_stattest_func: Optional[Union[PossibleStatTestType, Dict[str, PossibleStatTestType]]] = None
+
+    stattest_func: Optional[PossibleStatTestType] = None
+    cat_stattest_func: Optional[PossibleStatTestType] = None
+    num_stattest_func: Optional[PossibleStatTestType] = None
+    per_feature_stattest_func: Optional[Dict[str, PossibleStatTestType]] = None
```

Comment:
Maybe we can use `feature_stattest_func` and do not add a new parameter?

Reply:
Old options left there for backward compatibility as we decided to change it behaviour and rename all new options for simplicity.